### PR TITLE
tests, showcasing non working defaults for radios/dropdowns

### PR DESCRIPTION
**Pull request by benzkji**
if one removes the " what" from the radio (as value, and as default, and in the assertblock), it will work. At first sight, I dont know why it "doesnt" work.

this PR is a draft, not sure how I would incorporate this and/or refactor the current test suite.

### CODE REVIEW

File: form_designer/models.py [hunk 1]
```diff
@@ -257,6 +257,7 @@ def formfield(self):
         )
         if self.choices:
             kwargs["choices"] = self.get_choices()
+            kwargs["initial"] = slugify(self.default_value)
```
Comment (matthiask) [inline form_designer/models.py:260]: Yes, makes sense. 

```suggestion
            # The value of individual choices is slugified too.
            kwargs["initial"] = slugify(self.default_value)
```

**Review by matthiask — COMMENTED** [review]
Good find, thanks! I changed the base to run tests on GitHub actions etc. and added one suggestion, because it wasn't immediately clear to me why the `slugify()` is necessary.

**Comment by matthiask** [comment]
> this PR is a draft, not sure how I would incorporate this and/or refactor the current test suite.

Tests are passing now and you didn't mark the pull request as a draft. Do you want to clean it up or change anything? I don't think anything's missing (except for the CHANGELOG entry), but that's fine, really.

**Comment by benzkji** [comment]
ah, nice thanks. I didnt even know I could have marked is as a draft (in the github UI, right?)...so I just wrote it down ;-). Initial version for sure was a draft, only having failing tests. the solution was obvious, finally. I think it looks good, yes.

**Pull request merged**